apps/mdn/mdn-aws/docs/mdn-support.md

```diff
@@ -151,7 +151,47 @@ Example schedules:
 
 #### MDN backup cronjob
 
-- Docs for the `mdn-backup` cronjob are located [here](https://github.com/mozmeao/infra/tree/master/apps/mdn/utils/mdn_backup_cron).
+- Development docs for the `mdn-backup` cronjob are located [here](https://github.com/mozmeao/infra/tree/master/apps/mdn/utils/mdn_backup_cron).
+
+##### Rebuilding the backup cron image
+
+```
+cd infra/apps/mdn/utils/mdn_backup_cron/image
+./build.sh
+```
+
+##### MDN backup troubleshooting
+
+- To troubleshoot the backup cronjob, first delete the existing cronjob, then create a new pod using the `mdn-prod-backup-test.yaml` located in the `apps/mdn/utils/mdn_backup_cron/k8s/` directory of this repo. Once you've verified that the test pod is working correctly, recreate the cronjob.
+
+```
+    kubectl -n mdn-prod delete cronjob mdn-backup-prod
+    kubectl -n mdn-prod create -f infra/apps/mdn/utils/mdn_backup_cron/k8s/mdn-backup-test-pod.yaml
```

Comment:
If we're uploading tagged commit always, then we need to update ``mdn-backup-test-pod.yaml`` as well. Alternatively, ``build.sh`` could upload a tagged commit and ``:latest``, and ``mdn-backup-test-pod.yaml`` could always be ``:latest``.

Reply:
I've moved the `mdn-backup-test-pod.yaml` file to a Jinja2 template that's generated by the Makefile. Support docs updated to include new makefile target included in this PR.